app/components/gh-members-lab-setting.js

```diff
@@ -29,16 +29,22 @@ export default Component.extend({
     feature: service(),
     config: service(),
     mediaQueries: service(),
+    ghostPaths: service(),
 
     currencies: null,
 
     // passed in actions
     setMembersSubscriptionSettings() {},
+    setStripeConnectIntegrationTokenSetting() {},
 
     defaultContentVisibility: reads('settings.defaultContentVisibility'),
 
     stripeDirect: reads('config.stripeDirect'),
 
+    stripeConnectAuthUrl: computed('ghostPaths', function () {
```

Comment:
Is this correct? It _seems_ like it is!

Reply:
To be more specific, is the use of `computed` correct? And is it correct that this should be at the component level, and not higher?

Reply:
It doesn't need `computed`, can be a regular getter. The only thing you'd lose is built-in memoization of the result but that memoization is a bit confusing anyway because `ghostPaths` (dependent key) does not change over time and isn't tracked/used with `.set()`

Reply:
Updated and force pushed!